src/sprints/sprints.controller.ts

```diff
@@ -125,19 +145,28 @@ export class SprintsController {
     })
     @ApiResponse({
         status: HttpStatus.UNAUTHORIZED,
-        description: "User is not logged in",
+        description: "unauthorized access - user is not logged in",
         type: UnauthorizedErrorResponse,
     })
+    @ApiResponse({
+        status: HttpStatus.FORBIDDEN,
+        description: "forbidden - user does not have the required permission",
+        type: ForbiddenErrorResponse,
+    })
     @ApiParam({
         name: "meetingId",
         required: true,
         description: "voyage team Meeting ID (TeamMeeting/id)",
         example: 1,
     })
-    getMeetingById(@Param("meetingId", ParseIntPipe) meetingId: number) {
-        return this.sprintsService.getMeetingById(meetingId);
+    getMeetingById(
+        @Request() req: CustomRequest,
+        @Param("meetingId", ParseIntPipe) meetingId: number,
+    ) {
+        return this.sprintsService.getMeetingById(meetingId, req);
     }
 
+    @CheckAbilities({ action: Action.Create, subject: "Sprint" })
```

Comment:
I wonder if we should add new action `CreateSprintMeeting`, or `ManageSprintMeeting`
Action.Create, subject: "Sprint" sounds like create the actual sprints which should be only admin

Reply:
I was thinking instead of creating a new action  we can create a new subject  for sprintMeeting

Reply:
That's a better idea if you can make it work but I wasn't sure, I thought the subject has to be tied to a prisma model.  

We do have a team meeting model

Reply:
Yes we have a team meeting model and if we create a subject for it agenda and meeting related operations can be handled at one place.